smriprep/workflows/anatomical.py

```diff
@@ -742,70 +737,6 @@ def _pop(inlist):
     return inlist
 
 
-def _aseg_to_three():
```

Comment:
since we're planning on eventually reimplementing this, it may be best to leave these functions that include the logic and just not use them

Reply:
Restored.